swcc/swcc/models.py

```diff
@@ -396,44 +396,48 @@ def _load_data_spreadsheet(self):
 
         data = xls['data'].values
 
-        expected = ('shape_file', 'image_file')
         headers = next(data)
-        if headers[: len(expected)] != expected:
-            raise Exception(
-                'Unknown spreadsheet format in %r - expected headers to be %r, found %r'
-                % (file, expected, headers[: len(expected)])
-            )
+        for header in headers:
+            if not (header.startswith('shape_') or header.startswith('image_')):
```

Comment:
Before, we were allowing arbitrary columns after the first two.  Do we want to allow arbitrary non-shape / non-image columns (and just ignore them)?  Also, do we need to ensure there is at least one shape column?

Reply:
My understanding was that this sheet only contains shape and image data. I think it is a good idea to enforce at least one shape column though.